service/src/main/java/org/jboss/sbomer/service/feature/sbom/errata/dto/Errata.java

```diff
@@ -28,130 +30,105 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
-import lombok.experimental.SuperBuilder;
 import lombok.extern.slf4j.Slf4j;
 
 @Data
 @Slf4j
-@Builder(setterPrefix = "with")
-@NoArgsConstructor
-@AllArgsConstructor
 public class Errata {
```

Comment:
A general comment, not a requirement up to discussion. We could think about simplifying this and ensuring immutability of the content we received from ET by switching to Records for all the DTO's. Something like: https://dev.to/brunooliveira/practical-java-16-using-jackson-to-serialize-records-4og4

Reply:
mmm interesting, can be a good future refactoring indeed